bills-app/src/main/webapp/script.js

```diff
@@ -774,7 +774,7 @@ const getUpcomingBills = () => {
     fetch(url, {
         method: "GET", 
         headers: {
-            "X-API-Key": proPublicaApiKey
+            "X-API-Key": "/app.js"
```

Comment:
Why change this? I think this might be why there are all those errors about retrieving the bills printed to console.

Reply:
oh you're right! I forgot I changed this